src/sagas/compiledProjects.js

```diff
@@ -32,5 +33,8 @@ export function* validatedSource() {
 }
 
 export default function* () {
-  yield all([throttle(100, 'VALIDATED_SOURCE', validatedSource)]);
+  yield all([
+    throttle(100, 'VALIDATED_SOURCE', validatedSource),
+    takeEvery('APPLICATION_LOADED', validatedSource),
```

Comment:
This is fairly inside baseball, but I think we actually want to consume `PROJECT_CREATED` here—`APPLICATION_LOADED` will trigger different behavior in the projects saga depending on the initial application state (snapshot/gist ID in the URL, project rehydrated from local storage, etc.), and in some circumstances the saga will kick off a full validation. The case when it doesn’t do a full validation is exactly the case when it dispatches `PROJECT_CREATED`.

As a bonus, that’ll makes the console work right after you click “New Project”, which it currently does not—a closely related bug!.

Reply:
Oops! Right yes makes sense, tho didn't realize that re: "New Project"